store/smartAccount/createSmartAccount.ts

```diff
@@ -124,18 +96,26 @@ export async function createSmartAccount(username: string): Promise<{
       owners: [webAuthnAccount],
     });
 
-    // console.log('smartAccount', JSON.stringify(smartAccount));
-
     const address = await smartAccount.getAddress();
 
     console.log('address', address);
 
-    // Alert.alert('Smart account address', address);
-
     return { smartAccount, address };
   } catch (error: any) {
-    // TODO: error handling
-    // Alert.alert('Error creating smart wallet', error.message);
+    if (error.message && error.message.includes('Biometrics must be enabled')) {
+      Alert.alert(
+        'Your device is not enrolled to FaceID',
+        'If you are using a simulator, on the top menu bar, click on Features > Face ID > Enrolled',
+        [
+          {
+            text: 'Try again',
+            onPress: () => {
+              createSmartAccount(username, registrationOptions);
+            },
+          },
+        ]
+      );
+    }
```

Comment:
A side comment. This is where the passkey lib ought to call the permission request. Weird they don't

Reply:
I hope that's gonna be resolved by #36